docker-app/qfieldcloud/core/utils2/storage.py

```diff
@@ -112,13 +112,7 @@ def purge_old_file_versions(project: "Project") -> None:  # noqa: F821
     logger.info(f"Cleaning up old files for {project}")
 
     # Determine account type
-    account_type = project.owner.useraccount.account_type
-    if account_type == qfieldcloud.core.models.UserAccount.TYPE_COMMUNITY:
-        keep_count = 3
-    elif account_type == qfieldcloud.core.models.UserAccount.TYPE_PRO:
-        keep_count = 10
-    else:
-        raise NotImplementedError(f"Unknown account type {account_type}")
+    keep_count = project.owner.useraccount.account_type.storage_keep_versions
```

Comment:
The comment above is misleading, the account type is already determined in `useraccount`

Reply:
fixed